test/src/aave-integration-v3.spec.ts

```diff
@@ -0,0 +1,210 @@
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
+import { expect } from "chai";
+import { Contract, BigNumber } from "ethers";
+import { formatUnits, parseEther, parseUnits } from "ethers/lib/utils";
+import hre, { ethers } from "hardhat";
+
+import { Impersonate } from "../utils/utilities";
+
+describe("Test Token", function () {
+  let signer: SignerWithAddress;
+  let wrapper: Contract;
+  let wethGateway: Contract;
+  let token: Contract;
+  let pool: Contract;
+  let usdcAccount: SignerWithAddress;
+
+  let borrowAmount: BigNumber;
+
+  let usdc = "0x7F5c764cBc14f9669B88837ca1490cCa17c31607";
+  let weth = "0x4200000000000000000000000000000000000006";
+  let usdcHolder = "0xa3f45e619cE3AAe2Fa5f8244439a66B203b78bCc";
+
+  const MAX =  BigNumber.from(
+    "115792089237316195423570985008687907853269984665640564039457584007913129639935"
+  )
+
+  before(async () => {
+    [signer] = await ethers.getSigners();
+    usdcAccount = await Impersonate(usdcHolder);
+
+    const AaveWrapper = await ethers.getContractFactory("AaveWrapper", usdcAccount);
+    token = await ethers.getContractAt("IERC20", usdc, usdcAccount);
+    wrapper = await AaveWrapper.deploy();
+
+    pool = await ethers.getContractAt(
+      "IPoolV3",
+      "0x794a61358D6845594F94dc1DB02A252b5b4814aD",
```

Comment:
i think keeping this address constant in the file would be better. so that if we wish to change we can change at one place and it reflects  on the others